prow-configurations/config.yaml.tpl

```diff
@@ -13,29 +13,5 @@ presets:
     mountPath: /etc/service-account
     readOnly: true
 
-presubmits: # runs on PRs
```

Comment:
please add documentation how to organize our jobs, as you wrote here: https://app.zenhub.com/workspace/o/kyma-project/test-infra/issues/42

Reply:
Done